space-template.yaml

```diff
@@ -0,0 +1,166 @@
+---
+# These define the potential field types that ca be used in the fields of work
+# item types.
+field_types:
+  - &string string
+  - &translatable_string translatable_string
+  - &list list
+  - &timestamp timestamp
+  - &user user
+
+# A map of common strings that can be used to denote states of a work item
+common_work_item_states: &common_work_item_states
+  &state_new new:
+      en_US: "new"
```

Comment:
@maxandersen it depends. @michaelkleinhenz came up with the idea because WITs cannot be translated just like the UI. I agree that it can be massive but I just wanted to bring up this idea. We can maybe argue to specify a default locale (e.g. `en_US`) in the template and instead of making the string fields a map we make them a simple scalar string value that internally gets handled as if it was a map with `en_US` as the key for the locale.